### PR TITLE
Fix: recursive/glob support

### PR DESCRIPTION
**Pull request by calum-bird**
Changes:
`-r` is now a flag, not an argument
`file` argument changed to `path`, a string that can be either a file path, directory, or glob pattern
Logic to support the above in a natural way

This isn't quite ready for merging. Needs some testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +82,7 @@ python3 -m pip install homebrew-pypi-poet
 python3 -m pip install gptask_cli
 poet -f gptask_cli # Copy to ruby file
 ```
+
+```
+
+```
```
Comment (chitalian) [inline README.md]:
```suggestion
```

File: gptask_cli/__init__.py [hunk 2]
```diff
@@ -11,46 +11,58 @@ def check_file_staged_status(file, force):
...
+    """
+    if("*" in path):
+        return glob.glob(path, recursive=True)
+    elif os.path.isfile(path):
+        return [path]
+    elif(path[-1] == "/"):
+        path = path[:-1]
+    
+    # Recurse (or don't) through directory
+    return glob.glob(path + "/**/*" if is_recursive else path + "/*", recursive=True)
```
Comment (chitalian) [inline gptask_cli/__init__.py:26]: ahhh yeah - this makes a lot more sense lmao

**Comment by chitalian** [comment]
@calum-bird 
https://chat.openai.com/share/902cd378-3ebc-4e35-99ed-e63c7150c6ad

File: README.md (hunk 1 above)
Comment (calum-bird) [reply README.md]: Yeah fixing

File: gptask_cli/__init__.py (hunk 2 above)
Comment (calum-bird) [reply gptask_cli/__init__.py:26]: Yeah I think it's cleaner for sure

**Pull request merged**